Parsers/ASimWebSession/Parsers/ASimWebSessionCiscoMeraki.yaml

```diff
@@ -0,0 +1,228 @@
+Parser:
+  Title: Web Session ASIM parser for Cisco Meraki
+  Version: '0.1'
+  LastUpdated: Jun 20 2023
+Product:
+  Name: Cisco Meraki
+Normalization:
+  Schema: WebSession
+  Version: '0.2.6'
+References:
+- Title: ASIM Web Session Schema
+  Link: https://aka.ms/ASimWebSessionDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: Cisco Meraki Documentation
+  Link: https://documentation.meraki.com/
+Description: |
+  This ASIM parser supports normalizing Cisco Meraki logs to the ASIM Web Session normalized schema.
+ParserName: ASimWebSessionCiscoMeraki
+EquivalentBuiltInParser: _ASim_WebSession_CiscoMeraki
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let ActionLookup = datatable (action: string, DvcAction: string)  [
+    'allow', 'Allow',
+    'log', 'Allow',
+    'accept', 'Allow',
+    'block', 'Deny',
+    'deny', 'Deny',
+    'quarantine', 'Deny'
+  ];
+  let parser=(disabled: bool=false) {
+      union isfuzzy=true
+          (
+          meraki_CL
+          | project-rename LogMessage =  Message
+          ),
+          (
+          Syslog
+          | project-rename LogMessage =  SyslogMessage
```

Comment:
pls add watchlist filter logic for Syslog

Reply:
couldn't find this change, please comment

Reply:
Done